pkg/cmd/secret/set/set.go

```diff
@@ -87,6 +88,9 @@ func NewCmdSet(f *cmdutil.Factory, runF func(*SetOptions) error) *cobra.Command
 			# Set user-level secret for Codespaces
 			$ gh secret set MYSECRET --user
 
+			# Set repository-level secret for Dependabot
+			$ gh secret set MYSECRET --app dependabot
```

Comment:
We should also update the `long` description of the command to document that Dependabot secrets can be set.

Reply:
👍 https://github.com/cli/cli/pull/5134/commits/280ecb381dfc8f2f9088d5aa9adde556dbf6a280